workflow_miniscope/paths.py

```diff
@@ -21,7 +21,7 @@ def get_scan_image_files(scan_key):
     if tiff_filepaths:
         return tiff_filepaths
     else:
-        raise FileNotFoundError(f'No tiff file found in {sess_dir}')
+        raise FileNotFoundError(f'No .tif file found in {sess_dir}')
 
 
 def get_scan_box_files(scan_key):
```

Comment:
Should this be removed from the `miniscope` element/workflow entirely? 
(We can keep ScanImage, as that may still be used in many miniscope setup, but I don't think it's the case for `ScanBox`)

Reply:
Good catch.  Removed all `Scanbox` support from the `element` and `workflow`.